Makefile

```diff
@@ -33,6 +33,8 @@ CFLAGS += -Wundef -Iinclude -Ilib-lwip/src/include -I"$(LWIPOPTS_DIR)"
 
 NAME := lwip-core
 SRCS := $(LWIP_SRCS)
+# Disabling warnings from lib-lwip, as their code does not comply with these rules for now.
+LOCAL_CFLAGS += -Wno-char-subscripts -Wno-format-zero-length
```

Comment:
`-Wno-format-zero-length` arrises only in one place due to missing assertion description, IMO better fix would be to add this description

Reply:
I have mixed feelings. I don't want to change lib-lwip to much as it's ported code which aren't fixed upstream. On the other hand it's easy, one place change...

Reply:
Whatever you decide is fine with me